index.d.ts

```diff
@@ -1,69 +1,63 @@
-import { RenderProps } from 'react-intersection-observer'
+import React = require('react')
 
-declare module 'react-intersection-observer' {
```

Comment:
This is already the declaration for a module named `react-intersection-observer`, there's no need to explicitly declare it.

Also, because there was an `import` / `export` outside the `declare`, this wasn't actually declaring the module but... self-augmenting?